src/Api/EntityCustomFieldsApi.php

```diff
@@ -88,22 +89,20 @@ public function index(array $query = []): Response
 	 * @see https://api.sellsy.com/doc/v2/#operation/update-order-custom-fields
 	 * @see https://api.sellsy.com/doc/v2/#operation/update-credit-note-custom-fields
 	 */
-	public function update(CustomField $customField, array $query = []): Response
+	public function update(CustomFieldValue $customFieldValue): Response
 	{
-		$body = [
-			'id' => $customField->id
-		];
+		$body = $customFieldValue->toArray();
 		$response = $this->connection
 			->request("{$this->endpoint}/{$this->relatedEntity->id}/custom-fields")
-			->put([ $body + $query ]);
+			->put([ $body ]);
```

Comment:
Nearly there ! 

We just need to keep allowing passing `array $query = []` in the method (some additional params can be send, such as "verify") : 


```php
public function update(CustomFieldValue $customFieldValue, array $query = []): Response
{
    $body = $customFieldValue->toArray();

    $response = $this->connection
                    ->request($this->appendQuery("{$this->endpoint}/{$this->relatedEntity->id}/custom-fields", $query))
                    ->put([$body]);
                    
    // ...
}
```

The call to `appendQuery()` will automatically add any query param on the endpoint (e.g. `/companies/1/customer-fields?validate=true`).

Reply:
+ same on the `updateMany()` 🙏